content/en/blog/2021/migrate-alpha-security-policy/index.md

```diff
@@ -6,201 +6,177 @@ attribution: Yangmin Zhu (Google)
 keywords: [security,policy,migrate,alpha,beta,deprecate,peer,jwt,authorization]
 ---
 
-Istio beta security policy (`PeerAuthentication`, `RequestAuthentication` and `AuthorizationPolicy`) has been released long
-time ago and the alpha security policy (`MeshPolicy`, `Policy`, `ClusterRbacConfig`, `ServiceRole` and `ServiceRoleBinding`) has been
+Istio beta security policy (`PeerAuthentication`, `RequestAuthentication` and `AuthorizationPolicy`) has been released
+since release 1.5 and the alpha security policy (`MeshPolicy`, `Policy`, `ClusterRbacConfig`, `ServiceRole` and `ServiceRoleBinding`) has been
 deprecated and no longer supported starting 1.6.
 
 It is required to migrate the alpha security policy to the beta version before upgrading to Istio 1.6 and later.
-This blog is a tutorial to help customers who are still using the alpha security policy in old Istio to migrate to beta
-security policy and unblock the upgrade to Istio 1.6 and later.
+This blog is a tutorial to help customers who are still using the alpha security policy in the old version to migrate to
+the beta security policy and unblock the upgrade to Istio 1.6 and later.
 
-## Major changes in `v1beta1` security policy
+## Overview
 
-Please note there are non-backward compatible changes in the `v1beta1` policy. The major changes related to the migration
-are listed below:
+It is highly recommended migrating the security policy with the [canary upgrade](/docs/setup/upgrade/canary/).
```

Comment:
This is referring to canary the control plane, right? I think we can just briefly mention the control plane should be upgraded first as prerequisite and recommend to use [canary](link).

going into details of benefits seems a bit detour of what this doc is focused on.

Reply:
yeah, that makes a lot sense, updated, I also changed to recommend first upgrading to 1.5, the dual control plane is better for skip-upgrade even Istio do not support skip-upgrade.